assets/src/infrastructure/ui/layout/modal/styles.scss

```diff
@@ -1,23 +1,41 @@
 @import 'application/ui/styles/mixins/breakpoints.scss';
 
-.ee-modal .ee-modal {
-	&__header {
-		&.ee-modal__header {
-			[dir='rtl'] & {
-				padding-right: var(--ee-padding-extreme);
+.ee-modal {
+	.ee-modal {
+		&__header {
+			&.ee-modal__header {
+				[dir='rtl'] & {
+					padding-right: var(--ee-padding-extreme);
+				}
+
+				@include max782px {
+					padding-left: var(--ee-padding-smaller);
+					padding-right: var(--ee-padding-extreme);
+				}
 			}
+		}
 
+		&__body {
 			@include max782px {
 				padding-left: var(--ee-padding-smaller);
-				padding-right: var(--ee-padding-extreme);
+				padding-right: var(--ee-padding-smaller);
```

Comment:
GH got a bit confused here.
This is how it is on `EDTR/master`

![image](https://user-images.githubusercontent.com/1477453/87191627-798cb380-c2fd-11ea-9109-d3d8d6dd6a7e.png)